From 094cd349c8e33cc6119847f48b8ec86874a7c947 Mon Sep 17 00:00:00 2001
From: Robert Mustacchi <rm@fingolfin.org>
Date: Sat, 29 Jan 2022 19:59:17 +0000
Subject: [PATCH] 14458 scsi_hba_tgtmap_create(9F): various issues Reviewed by:
 Yuri Pankov <ypankov@tintri.com> Reviewed by: Toomas Soome <tsoome@me.com>
 Approved by: Rich Lowe <richlowe@richlowe.net>

---
 usr/src/man/man9f/scsi_hba_tgtmap_create.9f | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/usr/src/man/man9f/scsi_hba_tgtmap_create.9f b/usr/src/man/man9f/scsi_hba_tgtmap_create.9f
index 3ff2c46d31..808c189a27 100644
--- a/usr/src/man/man9f/scsi_hba_tgtmap_create.9f
+++ b/usr/src/man/man9f/scsi_hba_tgtmap_create.9f
@@ -10,8 +10,9 @@
 .\"
 .\"
 .\" Copyright 2019, Joyent, Inc.
+.\" Copyright 2022 Oxide Computer Company
 .\"
-.Dd June 03, 2019
+.Dd January 29, 2022
 .Dt SCSI_HBA_TGTMAP_CREATE 9F
 .Os
 .Sh NAME
@@ -173,8 +174,9 @@ argument.
 .Pp
 The
 .Fa dip
-argument should correspond to the HBA driver's device node or one of its
-iports.
+argument should correspond to one of the HBA driver's iports.
+The overall driver instance cannot be used here.
+Target maps are only supported on iports.
 .Pp
 The
 .Fa mode
@@ -198,7 +200,7 @@ argument to
 and use the
 .Fn scsi_hba_tgtmap_tgt_add
 and
-.Fn scsi_hba_tgtmap_tgt_destroy
+.Fn scsi_hba_tgtmap_tgt_remove
 functions.
 See the section
 .Sx Per-Address Reporting
-- 
2.40.1

